### PR TITLE
Feat/register signup optional params

### PR DESCRIPTION
**Pull request by ottony**
Accepts optional params on Api#register_signup

### CODE REVIEW

File: spec/incognia_spec.rb [hunk 1]
```diff
@@ -110,16 +111,48 @@ module Incognia
...
+          end
+
+          it_behaves_like 'receiving optional args', 'external_id' do
+            let(:opts) { { account_id: SecureRandom.uuid } }
+          end
+          it_behaves_like 'receiving optional args', 'external_id' do
+            let(:opts) { { external_id: SecureRandom.uuid } }
+          end
+          it_behaves_like 'receiving optional args', 'external_id' do
+            let(:opts) { { policy_id: SecureRandom.uuid } }
```
Comment (b0rba) [inline spec/incognia_spec.rb:153]: Should all of this be `'external_id'`?

**Review by b0rba — CHANGES_REQUESTED** [review]

**Comment by b0rba** [comment]
nit: PR title and body, and branch name has `feedback` but it is `sign-up`

**Comment by ottony** [comment]
> nit: PR title and body, and branch name has `feedback` but it is `sign-up`

True 🚬

File: spec/incognia_spec.rb (hunk 1 above)
Comment (ottony) [reply spec/incognia_spec.rb]: Nops, fixed

Comment (ottony) [reply spec/incognia_spec.rb]: :doggo-glad:

File: spec/incognia_spec.rb [hunk 2]
```diff
@@ -110,16 +111,48 @@ module Incognia
...
+                installation_id: installation_id,
+                **opts
+              )
+
+              expect(stub).to have_been_made.once
+            end
+          end
+
+          it_behaves_like 'receiving optional args', 'account_id' do
+            let(:opts) { { account_id: SecureRandom.uuid } }
```
Comment (alangalvino) [inline spec/incognia_spec.rb:147]: account_id and external_id are not necessarily UUIDs.

**Review by alangalvino — APPROVED** [review]

File: spec/incognia_spec.rb (hunk 2 above)
Comment (ottony) [reply spec/incognia_spec.rb:147]: It's just some random values for the test.

**Review by b0rba — APPROVED** [review]

**Pull request merged**